### PR TITLE
refactor: convert renderer classes to es6 classes

### PR DESCRIPTION
**Pull request by rachel-fenichel**
## The basics

- [x] I branched from develop
- [x] My pull request is against develop
- [x] My code follows the [style guide](https://developers.google.com/blockly/guides/modify/web/style-guide)

### Resolves

Part of #5860 and part of #2892

### Proposed Changes

Converts classes in the renderers directory to es6 classes.

### Reason for Changes

Move to ES6 classes to reduce hassle when converting classes to Typescript.

### Test Coverage
Normal tests, and after finishing each renderer I tested it in the advanced playground.

### Documentation

I added some new documentation where required by the compiler--generally jsdoc. I didn't need to make any major structural changes.

### CODE REVIEW

**Review by rachel-fenichel — COMMENTED** [review]
I added comments where I made non-automated changes, since you'll have a hard time finding them otherwise.

File: core/renderers/geras/renderer.js [hunk 1]
```diff
@@ -35,116 +34,118 @@ const {Theme} = goog.requireType('Blockly.Theme');
...
-Renderer.prototype.makePathObject = function(root, style) {
-  return new PathObject(
-      root, style,
-      /** @type {!ConstantProvider} */ (this.getConstants()));
-};
+  /**
+   * Create a new instance of the renderer's drawer.
+   * @param {!BlockSvg} block The block to render.
+   * @param {!BaseRenderInfo} info An object containing all
+   *   information needed to render this block.
```
Comment (alschmiedt) [inline core/renderers/geras/renderer.js:98]: The wrapping indentation seems a bit off. I believe it is 2 spaces when it should be 4.

**Review by alschmiedt — APPROVED** [review]

File: core/renderers/geras/drawer.js [hunk 2]
```diff
@@ -16,197 +16,200 @@
...
-/**
- * @override
- */
-Drawer.prototype.drawInlineInput_ = function(input) {
-  this.highlighter_.drawInlineInput(input);
+  /**
+   * @override
+   */
+  drawInlineInput_(input) {
+    this.highlighter_.drawInlineInput(/** @type {!InlineInput} */ (input));
```
[the remark this answers is not in the document]
Comment (rachel-fenichel) [reply core/renderers/geras/drawer.js:143]: It's a more specific type (`geras.InlineInput`).

File: core/renderers/geras/renderer.js (hunk 1 above)
Comment (rachel-fenichel) [reply core/renderers/geras/renderer.js]: Fixed.

**Pull request merged**